server/application/application.go

```diff
@@ -131,6 +131,11 @@ func (s *Server) Get(ctx context.Context, q *ApplicationQuery) (*appv1.Applicati
 	return s.appclientset.ArgoprojV1alpha1().Applications(s.ns).Get(*q.Name, metav1.GetOptions{})
 }
 
+// ListResourceEvents returns a list of event resources
+func (s *Server) ListResourceEvents(ctx context.Context, q *ApplicationResourceEventsQuery) (*v1.EventList, error) {
+	return s.kubeclientset.CoreV1().Events(s.ns).List(metav1.ListOptions{})
```

Comment:
I think this will return all events instead of events related to the specified resource. please fix it

Reply:
@alexmt right you are!  I believe this is now addressed.